tests/print-evaluated-rules/index.js

```diff
@@ -1,19 +1,19 @@
 const { ESLint } = require('eslint');
 const angularExtraRules = require('@angular-eslint/eslint-plugin/dist/configs/recommended--extra.json').rules;
-const angularRules = require('../angular').rules;
-const angularTemplateRules = require('../angular-template').rules;
+const angularRules = require('@ni/eslint-config-angular').rules;
+const angularTemplateRules = require('@ni/eslint-config-angular/template').rules;
 
 (async () => {
     const hasWarn = function (config) {
         return JSON.stringify(config).indexOf('"warn"') !== -1;
     };
 
     const eslint = new ESLint();
-    const configEslint = await eslint.calculateConfigForFile(`${__dirname}/../test/eslint/index.js`);
-    const configTypescript = await eslint.calculateConfigForFile(`${__dirname}/../test/typescript/index.js`);
-    const configTypescriptTypechecked = await eslint.calculateConfigForFile(`${__dirname}/../test/typescript-type-checking/index.js`);
-    const configAngular = await eslint.calculateConfigForFile(`${__dirname}/../test/angular/index.ts`);
-    const configAngularTemplate = await eslint.calculateConfigForFile(`${__dirname}/../test/angular/index.html`);
+    const configEslint = await eslint.calculateConfigForFile(`${__dirname}/../javascript/index.js`);
```

Comment:
This package is a known exception to the package dependency rule as it is a meta package comparing implementation details across packages. It is private and should not be published or duplicated. That will be captured in a README for the package.

Reply:
Added a README for the package.